### DISCUSSION TITLE
文本省略 Ellipsis ...能支持自定义吗？我想显示...查看更多

### DISCUSSION THREAD

**Discussion by reallifebogz** [tusen-ai/naive-ui#4000] [Q&A] [answered]
文本省略 Ellipsis ...能支持自定义吗？我想显示...查看更多
<img width="478" alt="image" src="https://user-images.githubusercontent.com/50833708/200280588-03bc930d-1ac1-4fbd-aab5-823718bf4c4c.png">

**Comment by XieZongChen** [comment]
提个issue吧

**Comment by h7ml** [comment, marked as answer]
[ellipsis](https://www.naiveui.com/zh-CN/os-theme/components/ellipsis) 组件https://github.com/tusen-ai/naive-ui/blob/bed18bc5487b7360750ddd6da6767abf29db5f12/src/ellipsis/src/Ellipsis.tsx#L50-L64 是基于css属性[-webkit-line-clamp](https://developer.mozilla.org/zh-CN/docs/Web/CSS/-webkit-line-clamp)来实现的，查了下属性表。并没有相关的将...替换为文本/标签相关文档。

但是 可以使用https://www.npmjs.com/package/ember-line-clamp 该库来实现自定义查看更多功能
